scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -382,20 +385,35 @@ <h3 class="header">Institute Settings</h3>
             </td>
             <td><!-- CADD score -->
               {% if variant.cadd_score %}
-                <span class="badge bg-secondary">{{ variant.cadd_score }}</span>
+                <span class="badge bg-secondary">{{ variant.cadd_score|round(2) }}</span>
               {% else %}
                 -
               {% endif %}
             </td>
             <td><!-- Rank score -->
-              <a href="#"><span class="badge rounded-pill bg-secondary">
+              <span class="badge rounded-pill bg-secondary">
               {{ variant.rank_score }}
               {% if variant.category == 'sv' %}
                 (v. {{case.sv_rank_model_version or 'na'}})
               {% else %}
                 (v. {{case.rank_model_version or 'na'}})
               {% endif %}
-              </span></a>
+              </span>
+            </td>
+            <!-- Revel scores -->
+            {% for score in [variant.revel, variant.revel_score] %}
+            <td>
+              {% if score %}
+                <span class="badge rounded-pill bg-secondary">
+                  {{score}}
+                </span>
+              {% else %}
+                -
```

Comment:
Ah, these are not very visible for dark mode. I guess they are also not very important to view in the first place. But, to keep the same experience in light and dark, perhaps keep the span for both existing and non existing?

Reply:
![Screenshot 2024-01-10 at 09 48 38](https://github.com/Clinical-Genomics/scout/assets/758570/2efcd913-0439-4873-8eb9-bfdc4ef1e648)

Reply:
Fixed as well:

<img width="124" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/25f3614d-c693-411a-8e0c-329fc6867d11">